h/search/docs/selector.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from elasticsearch_dsl import (InnerDoc,
+                               Keyword,
+                               Long,
+                               Text,
+                               )
+
+
+class Selector(InnerDoc):
```

Comment:
We could chose to put Selector and Target in annotation.py since they are InnerDocs of Annotation. Either way when creating an Annotation you need to create a Target and Selector object and pass it in upon Annotation obj creation.